src/dmd/globals.d

```diff
@@ -561,6 +561,8 @@ struct Loc
     const(char)* filename; // either absolute or relative to cwd
     uint linnum;
     uint charnum;
+    version (DMDLIB)
```

Comment:
The version should be something like LocOffset, because that's what it is. DMDLIB has no particular connection to offset, and should be independent, although it can be set based on DMDLIB.